tools/integration_tests/managed_folders/admin_permissions_test.go

```diff
@@ -33,18 +35,171 @@ import (
 // Boilerplate
 // //////////////////////////////////////////////////////////////////////
 
+const (
+	MoveFile       = "moveFileAdminPerm"
+	MoveDestFile   = "moveDestFileAdminPerm"
```

Comment:
MoveDestFile is unused. I also think you can use just two variables for source and destination here.

Reply:
It used the same for the destination. In the case of the source, it gives an intermittent error of `Error in stating file` as we create resources before a test and delete them after running the test. It is taking time to reflect. Adding sleep is working, but it will increase run time; that's why I used different sources.